app/javascript/mastodon/features/ui/index.js

```diff
@@ -128,11 +133,18 @@ class UI extends React.PureComponent {
         </ColumnsArea>
       );
     } else {
+      // TODO: Better check & error handling
```

Comment:
Why not just initialize `staticColumns` with `null` and test it is `null`? I guess you did so in favor of `const`, but `const` does not really make difference in this case because it just eliminates a possibility (reassignment) to get broken while there are many other similar possibilities.